jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunMojo.java

```diff
@@ -144,7 +144,7 @@ private void startScanner()
     {
         if (scan < 0)
         {
-            getLog().info("Redeployment not enabled");
+            getLog().info("Redeployment Disabled; Default Scan Interval = -1 ; Change Value to <scan>0<scan> for Manual or <scan>${Seconds}<scan> for automatic  ");
```

Comment:
Thanks for the pull request, I understand you're looking for more configuration info (which is in the doco btw). I think this log line might be a little wordy, how about something a bit more in line with the other configuration summaries the plugin produces, that is a little more succinct like:  

"scan=-1 redeploy disabled (scan=0:redeploy on ENTER; scan > 0:automatic redeploy check sec)"

Reply:
I think yours is better.
I would like to see  the word default
default scan=-1 redeploy disabled; scan=0:redeploy on <ENTER>; scan > 0:automatic redeploy check sec

Reply:
default mode scan=-1:redeploy disabled;  scan=0:redeploy manual on ENTER;  scan > 0:redeploy automatic

I am happy with this. The default also may alert the user that there is other functionality available. 
Hot Deploy is a very useful feature and needs to be brought to the forefront attention of the user.
The user being a developer he will know implications into with word  default.
I did not suggest a changing the default behaviour because I am sure you had a good reason  for it. 
Personally my preference is default mode to be manual deploy mode.
may be add the word "mode" also with default ?